<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileBrowserClass</class>
 <widget class="QMainWindow" name="FileBrowserClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>317</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Сравнение состояний реактора</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>621</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="CommonLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>621</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Выберете 2 файла для сравнения состояний:</string>
    </property>
   </widget>
   <widget class="QLabel" name="File1Label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Первое состояние:</string>
    </property>
   </widget>
   <widget class="QLabel" name="File2Label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Второе состояние:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="File1Button">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>557</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Обзор...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="File2Button">
    <property name="geometry">
     <rect>
      <x>557</x>
      <y>60</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Обзор...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Compare">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>250</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Сравнить</string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Exit">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>250</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Отмена</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="File1lineEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>411</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="File2lineEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>411</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <zorder>textBrowser</zorder>
   <zorder>CommonLabel</zorder>
   <zorder>File1Label</zorder>
   <zorder>File2Label</zorder>
   <zorder>File2Button</zorder>
   <zorder>Compare</zorder>
   <zorder>Exit</zorder>
   <zorder>File1lineEdit</zorder>
   <zorder>File2lineEdit</zorder>
   <zorder>File1Button</zorder>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QStatusBar { 
					border-top-style: outset; 
					border-top-width: 1px; 
					border-top-color: #828282; 
					}</string>
   </property>
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="filebrowser.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>Exit</sender>
   <signal>clicked()</signal>
   <receiver>FileBrowserClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>400</x>
     <y>404</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
